### DISCUSSION TITLE
Pythia Cookbooks Planning

### DISCUSSION THREAD

**Discussion by mgrover1** [ProjectPythia/projectpythia.github.io#219] [Ideas]
Reviving #170 ! Our Education Working Group Discussion on March 17, 2022 will be dedicated to generating some plans for creating Pythia Cookbook prototypes.

This is a place to chat asynchronously prior to the meeting, and gather ideas how we plan on moving forward.

**Comment by brian-rose** [comment]
Note that there's some discussion fodder already over at https://github.com/ProjectPythia/pythia-cookbook/issues/1

**Comment by r-ford** [comment]
How is the [Pangeo Gallery](http://gallery.pangeo.io/) related to Pythia Cookbooks? I see that it's linked in our own gallery, but is that the sort of format we have in mind for the cookbooks?

**Reply by mgrover1** [reply]
> How is the [Pangeo Gallery](http://gallery.pangeo.io/) related to Pythia Cookbooks? I see that it's linked in our own gallery, but is that the sort of format we have in mind for the cookbooks?

Great question! I envision these cookbooks being more than just examples... I mention it in more detail in #170, but the key here is that the [Foundations](https://foundations.projectpythia.org/landing-page.html) content is linked throughout the content, in addition to a full narrative from initial data ingestion through visualization.

We can really dig into what foundational skills required, and provide users the additional specialized content (ex. Dask, Cloud Data, Satellite Imagery) that is required to complete different workflows... 

I hope that makes sense! This will be good to bring up at the discussion too.

**Reply by r-ford** [reply]
Yeah, that makes sense. I saw that CMIP6 was mentioned in [this issue](https://github.com/ProjectPythia/pythia-cookbook/issues/1), so maybe we would want to reuse and modify the tutorials in the [CMIP6 gallery](http://gallery.pangeo.io/repos/pangeo-gallery/cmip6/), which also has some Dask.

**Comment by clyne** [comment]
A few comments on some things that are probably pretty obvious to discuss:

1. Creating a template to ease adding new cookbooks
2. Cookbooks could be focused on performance (e.g. Dask) or require larger data sets. With Foundations, everything is intended to be "runnable", either via binder or by downloading to a local machine. We may want to relax this for cookbooks
3. How should cookbooks be organized so they are findable? Can we make use of our awesome Gallery filtering?

**Comment by mgrover1** [comment]
I put together a graphic for the "Radar Cookbook" discussion we are having at ARM/DOE today. It highlights how we can reference the new materials, while generating project/data specific information.

<img width="1334" alt="Screen Shot 2022-03-08 at 9 31 42 AM" src="https://user-images.githubusercontent.com/26660300/157270536-41b51e40-1451-4ca8-9d94-b7f899025298.png">

I will be taking notes and updating discussion points here as well.

**Comment by brian-rose** [comment]
Very cool, @mgrover1!

It will be interesting to see what patterns emerge in the cross-references as the Cookbook collection takes root. That should help highlight what Foundations is covering well and what is missing.
